src/views/catalog-product-details.vue

```diff
@@ -874,7 +874,11 @@ export default defineComponent({
         if (Object.keys(presellingJob).length === 0 || !presellingJob.runTime) {
           this.poSummary.header = this.$t("Pre-sell processing disabled");
         } else {
-          this.poSummary.header = this.$t("Removing from in", { categoryName, addingTime: this.timeTillJob(presellingJob.runTime) });
+          if(this.timeTillJob(presellingJob.runTime)?.includes('in')){
```

Comment:
Find some other way to identify the past time.
Also check with @Dhiraj1405 for specific messages in the two cases.